zingolib/src/commands.rs

```diff
@@ -1513,7 +1471,6 @@ pub fn get_commands() -> HashMap<&'static str, Box<dyn Command>> {
         ("save", Box::new(SaveCommand {})),
         ("quit", Box::new(QuitCommand {})),
         ("list", Box::new(TransactionsCommand {})),
-        ("notes", Box::new(NotesCommand {})),
```

Comment:
please mention that this function is removed in a changelog

Reply:
Mentioned move to test-only.